lib/ClassUtils.lua

```diff
@@ -25,16 +25,20 @@ function ClassUtils.makeClass(name, constructor)
 end
 
 function ClassUtils.makeConstructedClass(name, constructor)
```

Comment:
makeConstructedClass and makeClassWithConstructor

Reply:
The point is a "constructed" class is one where the instance is already created with the fields passed in to new before the constructor is called. I'll update the "constructor" parameter name and documentation to explain